connectors/config.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+
+import copy
+
+from envyaml import EnvYAML
+
+
+class ConfigNotLoadedError(Exception):
+    pass
+
+
+class Config:
+    _yaml = None
+
+    @classmethod
+    def load(cls, config_file):
+        cls._yaml = EnvYAML(config_file)
+
+    @classmethod
+    def get(cls):
```

Comment:
Are there other cases where only a specific part of the configuration is needed? Like `Config.get("elasticsearch")` or `Config.get("service")`?

Reply:
I can add this function, but it's not in use for now.